eng/scripts/splittestdependencies/README.md

```diff
@@ -0,0 +1,43 @@
+# Test dependency Splitting Strategy
+## Problems
+Today, most of the Azure SDKs implemented in our repo have a dependency on the core library. In order to guarantee the changes on core do not break the SDKs depend on it, our core PR validation pipeline currently runs all tests under `sdk` folder. As the onboarding packages keep growing, the test steps of pipeline usually take nearly an hour to finish. To achieve the purpose of speeding up the PR validation, we decided to move on to the test splitting task.
+
+## Existing workflow
+Before we run the tests, we have a step to determine the testing matrix (OS image, pool, .NET framework etc.). For each of the testing environment, we check which is the service triggers the pipeline. If it is service other that core, then we run the tests under the service only. If it is core library, then we will run all tests under `sdk` folder. There are more than 400 packages in total as of today, so even we execute them in parallel, it still takes long while to get them done. 
+
+## New workflow
+In order to speed up the pipeline, we have to answer two questions here:
+
+1. Is it necessary to run all tests under `sdk`? Can we fetch the tests which have a dependency on core?
```

Comment:
I assume this includes transitive dependencies? i.e. any T2 library tests will still be run as part of Core pipeline.

Reply:
The dependencies I pulled out has a direct dependency on core. Do you have any package example for the transitive dependencies? I can check over in my local if we include it or not.